test/test_markdown_renderer.py

```diff
@@ -577,8 +576,8 @@ def test_wordwrap_paragraph_in_list(self):
                 "1. List item\n"
                 "2. A second list item\n"
                 "   including:\n"
-                "   * Nested list. This is\n"
-                "     a continuation line\n"
+                "   * Nested list. This is a\n"
+                "     continuation line\n"
```

Comment:
I have no clue why this has changed.

Reply:
Yeah, I struggled to find the cause this change as well for a while, then I found it in the calculation of `max_child_line_length` - see my fix in the code review commit.